serializer.rst

```diff
@@ -66,17 +74,32 @@ As well as the following normalizers:
 * :class:`Symfony\\Component\\Serializer\\Normalizer\\ArrayDenormalizer` to
   denormalize arrays of objects using a format like `MyObject[]` (note the `[]` suffix)
 
+As well as the following encoders:
+
+* JSON: :class:`Symfony\\Component\\Serializer\\Encoder\\JsonEncoder`
+* XML: :class:`Symfony\\Component\\Serializer\\Encoder\\XmlEncoder`
+
+These normalizers need to be loaded manually:
+
+* :class:`Symfony\\Component\\Serializer\\Normalizer\\GetSetMethodNormalizer` a
```

Comment:
I would not mention this normalizer here. We'll maybe deprecate it at some point.